versions.json

```
@@ -1,6 +1,6 @@
 {
   "abigen": "v1.10.25",
-  "foundry": "617dfc28cb8206a0003edcf73a6f1058adaef740",
+  "foundry": "dbc48ead1044066a3e12c796fca9dc077f5913fe",
```

Nightlies are only persisted for 3 nights but the first one of each month is persisted. Sometimes its a bit quirky and the one from the day after gets persisted (e.g. if a release fails). I think https://github.com/foundry-rs/foundry/releases/tag/nightly-d94e3c631e2da7756af46c70f8f58b75563b7013 will be persisted, so my suggestion is wait another day or two confirm that, then upgrade to that version instead, since this one will be pruned soo

---

> my suggestion is wait another day or two confirm that, then upgrade to that version instead, since this one will be pruned too.

If we think this one will likely persist, would it be reasonable to just make the change here now, and if fix if we notice it doesn't get persisted?

```suggestion
  "foundry": "d94e3c631e2da7756af46c70f8f58b75563b7013",
```

---

That's ok with me, just might be more churn/overhead if we are wrong about the persisted tag, since a new ci-builder release is needed with each bump

---

oh, good point, OK I'll sit on this for now.